src/Tribe/Main.php

```diff
@@ -2272,6 +2272,157 @@ public function eventQueryVars( $qvars ) {
+				'tribe-events-calendar' => $this->pluginDir . 'lang/',
+			);
+
+			// If WPML exists we treat the multiple languages
+			if ( function_exists( 'icl_get_languages' ) ) {
+				global $sitepress;
+
+				// Grab all languages
+				$_languages = $sitepress->get_active_languages();
+
+				// Query the Current Language
+				$clang = $sitepress->get_current_language();
+				$clocale = $sitepress->get_locale( $clang );
+
+				foreach ( $_languages as $lang ) {
+					$languages[] = $sitepress->get_locale( $lang['code'] );
+				}
+
+				// Prevent Duplicates and Empty langs
+				$languages = array_filter( array_unique( $languages ) );
+
+				// Get the strings on multiple Domains and Languages
+				$bases = $this->get_i18n_strings( $bases, $languages, $domains );
+			}
+
+			if ( 'regex' === $method ){
+				foreach ( $bases as $type => $base ) {
+					$base = array_map( 'sanitize_title_with_dashes', $base );
+					$bases[ $type ] = '(' . implode( '|', $base ) . ')';
+				}
+			}
+
+			return (object) $bases;
+		}
+
+		/**
+		 * Get all possible translations for a String based on the given Languages and Domains
+		 *
+		 * WARNING: This function is slow because it deals with files, so don't overuse it!
+		 *
+		 * @todo Include support for the `load_theme_textdomain` + `load_muplugin_textdomain`
+		 *
+		 * @param  array  $strings          An array of strings (required)
+		 * @param  array  $languages        Which l10n to fetch the string (required)
+		 * @param  array  $domains          Possible Domains to re-load
+		 * @param  string $default_language The default language to avoid re-doing that
+		 *
+		 * @return array                    A multi level array with the possible translations for the given strings
+		 */
+		public function get_i18n_strings( $strings, $languages, $domains = array( 'tribe-events-calendar' ), $default_language = 'en_US' ) {
+			foreach ( $languages as $language ) {
+				// Prevent re-loading the default language
+				if ( $language === $default_language ){
+					continue;
+				}
+
+				foreach ( (array) $domains as $domain => $file ) {
+					// Configure the language
+					$this->_locale = $language;
+					add_filter( 'locale', array( $this, '_set_locale' ) );
+
+					// Reload it with the correct language
+					unload_textdomain( $domain );
+
+					if ( 'default' === $domain ) {
+						load_default_textdomain();
+					} else {
+						load_plugin_textdomain( $domain, false, $file );
+					}
+
+					// Loop on the strings the build the possible translations
+					foreach ( $strings as $key => $value ) {
+						$value = is_array( $value ) ? reset( $value ) : $value;
+						if ( ! is_string( $value ) ) {
+							continue;
+						}
+
+						// Make sure we have an Array
+						$strings[ $key ] = (array) $strings[ $key ];
+
+						// Grab the possible strings for Default and Any other domain
+						if ( 'default' === $domain ) {
+							$strings[ $key ][] = __( $value );
+							$strings[ $key ][] = __( strtolower( $value ) );
```

Comment:
On this case the Literals will be _"defined"_ elsewhere. I'm not sure where I will do it.